Source/Checkers.cs

```diff
@@ -84,7 +85,8 @@ public static bool IsUnityCompatible()
         }
 
         // Version of the compatibility checker itself.
-        private static int _version = 5;
+
```

Comment:
Isn't it already separated though?  This newline is between the comment and the field.  It just seemed odd to me.